.gitignore

```diff
@@ -373,3 +373,5 @@ docs/\.ionide/
 \.ionide/
 
 
+
+NotebookExamples/csharp/.ipynb_checkpoints/
```

Comment:
Would it make more sense to ignore any `.ipynb_checkpoints` directory?

See
[https://github.com/github/gitignore/blob/eb14c04c3596082f0d16a543d54b1fe1608c6498/community/Python/JupyterNotebooks.gitignore#L5](https://github.com/github/gitignore/blob/eb14c04c3596082f0d16a543d54b1fe1608c6498/community/Python/JupyterNotebooks.gitignore#L5)
[https://github.com/github/gitignore/blob/c1faae44aebe587bbb90d60e21ca928e0116e8d4/Python.gitignore#L77](https://github.com/github/gitignore/blob/c1faae44aebe587bbb90d60e21ca928e0116e8d4/Python.gitignore#L77)